### PR TITLE
Finished Climber Subsystem file

### LINKED ISSUE #13: Climber Subsystem
Create one Climber subsystem class to represent the climber on the bot. 

Methods 
- lockSet set a piston in and out true being in false being out
- lockGet get the state of the climber locking piston return a boolean
- speedSet set the climber motor speed intakes a double as a parameter
- speedGet get the speed of the climber returns speed as a double

### PR DESCRIPTION
**Pull request by Spenceroni-Pepperoni**
Closes #13

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+
+  }
+
+  /**
+   * Sets the Climer motor speed to go up
+   */
+  public static void speedSetUp() {
```
Comment (SixtyFiveOhTwo) [inline src/main/java/frc/robot/subsystems/Climber.java]: Please make one speedSet method that takes in a double as a parameter

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+public class Climber extends SubsystemBase {
+  /** Creates a new Climber. */
+  static TalonSRX climberMotor;
+  static DoubleSolenoid cSolenoid;
+  static boolean pistonExtended = false;
+
+  public Climber() {
+    climberMotor = new TalonSRX(Constants.CLIMBER_MOTOR_ID);
+    cSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, 5, 6);
```
Comment (SixtyFiveOhTwo) [inline src/main/java/frc/robot/subsystems/Climber.java]: Please change this to get  the solenoid port numbers from constants

**Review by SixtyFiveOhTwo — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+   */
+  public static void speedSetUp() {
+    climberMotor.set(TalonSRXControlMode.PercentOutput, 0.5);
+  }
+
+  /**
+   * Sets the Climer motor speed to go down
+   */
+  public static void speedSetDown() {
+    climberMotor.set(TalonSRXControlMode.PercentOutput, 0.5);
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/Climber.java]: If this is "down" and the prior is "up," why are both assigning the same value?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+  /**
+   * Locks the climber in place
+   */
+  public static void lockSet() {
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/Climber.java]: This looks like a toggle. If it is intended to behave as a toggle, name and document it as such.

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    }
+  }
+
+  /**
+   * Gets the position of the piston and checks if locked
+   * 
+   * @param isPistonExtended same value as climberMotor
+   * @return The state of the piston
+   */
+  public static boolean lockGet(boolean isPistonExtended) {
```
Comment (liam-a-wms) [inline src/main/java/frc/robot/subsystems/Climber.java]: Says it's a get, is documented as a get, but behaves like a set. If this is supposed to be a set, please name and document it as such. If it's supposed to be a get, please update the behavior to be that of a get.

**Review by liam-a-wms — CHANGES_REQUESTED** [review]

**Comment by Spenceroni-Pepperoni** [comment]
Added Changes

**Review by liam-a-wms — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 6]
```diff
@@ -26,15 +26,21 @@
     public static final int ID_RIGHT_PRIMARY = 10;
     public static final int ID_RIGHT_SECONDARY = 11;
 
+    // Clmiber Motor Can IDs
```
Comment (Gliu1211) [inline src/main/java/frc/robot/Constants.java]: Correct spelling

**Review by Gliu1211 — APPROVED** [review]
looks good

**Review by SixtyFiveOhTwo — APPROVED** [review]

**Pull request merged**